src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -948,6 +948,11 @@ def run_datamover(self, test_desc=None,
         if test_desc is not None:
             self.log.info("Running %s: %s", self.tool, test_desc)
 
+        env = EnvironmentVariables.from_list(
+            self.params.get("env_vars", "/run/{}/*".format(self.tool.lower()), []))
+        if "D_LOG_FILE" not in env:
+            env["D_LOG_FILE"] = get_log_file("{}.log".format(self.tool.lower()))
```

Comment:
I don't expect this logic to fail, but I didn't verify on other tests. If we want to land with this logic, I think we should also run some sanity tests:
```
Test-tag: datamover,vm
```